src/Tickets/Commerce/Admin/Orders_Page.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Handles displaying of orders page and sub menu.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+
+namespace TEC\Tickets\Commerce\Admin;
+
+use TEC\Tickets\Commerce\Order;
+use Tribe\Admin\Pages;
+
+/**
+ * Class Orders_Page.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+
+/**
+ * Manages the admin settings UI in relation to ticket configuration.
```

Comment:
Does this need to be updated?

Reply:
Good call. That doesn't even need to be there.